src/validation/beneficial.owner.individual.validation.ts

```diff
@@ -20,6 +20,9 @@ export const beneficialOwnerIndividual = [
     .isLength({ max: 160 })
     .withMessage(ErrorMessages.MAX_LAST_NAME_LENGTH)
     .matches(VALID_CHARACTERS).withMessage(ErrorMessages.LAST_NAME_INVALID_CHARACTERS),
+
+  ...date_of_birth_validations,
```

Comment:
just moved the order the validators run so that the error messages are in field order in teh box at top of page